### PR TITLE
[RNMobile] Prevent bold headings after autocorrection

### PR DESCRIPTION
**Pull request by twstokes**
## Description
Related PRs:
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/4557
- https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1334

## Testing Instructions
See Gutenberg Mobile PR for testing instructions:
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/4557

## Types of changes

Bug fix

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal). 
- [x] I've updated related schemas if appropriate.

### CODE REVIEW

File: packages/react-native-editor/ios/Podfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ project 'GutenbergDemo.xcodeproj'
 require_relative '../../../node_modules/react-native/scripts/react_native_pods'
 require_relative '../../../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
+# ONLY FOR TESTING, IT SHOULD BE DELETED ONCE THE AZTEC PR IS MERGED!
+pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => '8d72edcde64eb2a05175ae877fad97985153c26c'
+
```
Comment (fluiddot) [inline packages/react-native-editor/ios/Podfile]: Heads up that this should be removed before merging, as `react-native-editor` should point to an Aztec release.

**Review by fluiddot — APPROVED** [review]
Approved via https://github.com/wordpress-mobile/gutenberg-mobile/pull/4557#pullrequestreview-880363796.

**NOTE:** An Aztec-iOS release should be done before merging this PR, and it should be updated in the different places where it's referenced (e.g. GB-mobile, Gutenberg and WP-iOS).

File: packages/react-native-editor/ios/Podfile (hunk 1 above)
Comment (twstokes) [reply packages/react-native-editor/ios/Podfile]: Done. 👍

**Review by fluiddot — APPROVED** [review]
Approved via https://github.com/wordpress-mobile/gutenberg-mobile/pull/4557#pullrequestreview-881461749.

**Pull request merged**